Sources/GDSAnalytics/Events/TaxonomyType.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+public struct TaxonomyType {
+    public enum FirstLevel: String {
+        case oneLogin = "one login mobile application"
```

Comment:
nope!! not here! 
This file is all business logic, it needs to live in the app and get passed in

Reply:
Ah! I chatted with Amish about this and we knew we couldn't set it in common but thought we could have the types - will remove this